### PR TITLE
Cinnamon Survey 20220325

### PR DESCRIPTION
**Pull request by Cyanoxygen**
Topic Description
-----------------
- Update cinnamon DE to the latest version
- Revert `caribou` package drop - cinnamon can not launch without this

Package(s) Affected
-------------------

- `groups/cinnamon`
- `caribou`

Security Update?
----------------

No

Build Order
-----------

- `caribou`
- `groups/cinnamon`

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [ ] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by Cyanoxygen** [comment]
Testing
------

The updated Cinnamon DE is running well on my test machine (a Virtualbox VM):
image

File: extra-gnome/caribou/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,9 @@
+PKGNAME=caribou
+PKGDES="A text entry and UI navigation application (on-screen keyboard)"
+PKGSEC=gnome
+PKGDEP="at-spi2-atk clutter dconf gtk-3 libgee libxklavier pyatspi pygobject-3"
+BUILDDEP="gtk-2 intltool docbook-xsl"
+
+
+NOLTO=1
```
Comment (WhiredPlanck) [inline extra-gnome/caribou/autobuild/defines]: Why?

**Review by WhiredPlanck — CHANGES_REQUESTED** [review]

File: extra-gnome/caribou/autobuild/defines (hunk 1 above)
Comment (Cyanoxygen) [reply extra-gnome/caribou/autobuild/defines]: Oh, probably was left over while I experimenting building this package.

**Review by MingcongBai — DISMISSED** [review]

**Review by WhiredPlanck — DISMISSED** [review]

File: extra-gnome/caribou/autobuild/build [hunk 2]
```diff
@@ -0,0 +1,19 @@
+abinfo "Running autogen.sh..."
+NOCONFIGURE=1 $SRCDIR/autogen.sh
+
+abinfo "Unset PYTHON to disable Python 2"
```
Comment (MingcongBai) [inline extra-gnome/caribou/autobuild/build]: You could use “PYTHON=“ in AUTOTOOLS_AFTER to achieve the same effect.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**